tsfile/src/test/java/org/apache/iotdb/tsfile/read/common/PathTest.java

```diff
@@ -68,16 +79,33 @@ public void addHeadPath() throws Exception {
   }
 
   @Test
-  public void replace() throws Exception {
-    Path src = new Path("a.b.c");
-    Path rep1 = new Path("");
-    Path rep2 = new Path("d");
-    Path rep3 = new Path("d.e.f");
-    Path rep4 = new Path("d.e.f.g");
-    testPath(Path.replace(rep1, src), "a.b", "c", "a.b.c");
-    testPath(Path.replace(rep2, src), "d.b", "c", "d.b.c");
-    testPath(Path.replace(rep3, src), "d.e", "f", "d.e.f");
-    testPath(Path.replace(rep4, src), "d.e.f", "g", "d.e.f.g");
+  public void addHeadPathWithQuote() {
+    Path desc = new Path("a.b.\"c\"");
+    Path head = new Path("d.e");
+    Path head1 = new Path("");
+    testPath(Path.addPrefixPath(desc, head), "d.e.a.b", "c", "d.e.a.b.\"c\"");
+    testPath(Path.mergePath(desc, head1), "a.b", "c", "a.b.\"c\"");
   }
 
+  //  @Test
+//  public void replace() throws Exception {
+//    Path src = new Path("a.b.c");
+//    Path rep1 = new Path("");
+//    Path rep2 = new Path("d");
+//    Path rep3 = new Path("d.e.f");
+//    Path rep4 = new Path("d.e.f.g");
+//    testPath(Path.replace(rep1, src), "a.b", "c", "a.b.c");
+//    testPath(Path.replace(rep2, src), "d.b", "c", "d.b.c");
+//    testPath(Path.replace(rep3, src), "d.e", "f", "d.e.f");
+//    testPath(Path.replace(rep4, src), "d.e.f", "g", "d.e.f.g");
+//  }
+
+  @Test
+  public void temp() {
+    Path src = new Path("a.b.c");
+    String s = "hai \" hello\"";
+    String[] ss = s.split("\"");
+    System.out.println(ss.length);
+    System.out.println(Arrays.toString(ss));
```

Comment:
The test does not seem to test anything, remove or complete it.

Reply:
Thanks for your suggestion. These codes are removed.